### ISSUE TITLE
Correct dimension for basket cell & OLM in CA3 example?

### ISSUE THREAD

**Issue by pgleeson** [suny-downstate-medical-center/netpyne#802]
The CA3 example network looks like a great one to highlight more on OSB (https://github.com/suny-downstate-medical-center/netpyne/tree/development/examples/CA3model_3pops), but the dimensions of the basket (white) and olm cells (green), compared to the more normal looking ca3 pyr cell (purple). Is this intentional @salvadord @vvbragin, i.e. have these cells been given larger dims to compensate for no dends etc? I've reduced the numbers of cells in the network for clarity.

<img width="1315" alt="Screenshot 2023-12-15 at 11 46 06" src="https://github.com/suny-downstate-medical-center/netpyne/assets/1556687/ec039420-fbe9-493e-983c-a63049986df7">

**Comment by salvadord** [comment]
@urdapile can you have a look at this? I think you implemented this model right?

**Comment by urdapile** [comment]
yes, sure.

**Comment by urdapile** [comment]
@pgleeson @salvadord, the model is the NetPyNE replication of the NEURON model developed by Sam Neymotin et al (https://doi.org/10.1523/JNEUROSCI.0501-11.2011), available at ModelDB (https://modeldb.science/139421). In this model, I don't know the reason, the inhibitory cells were assigned an area of 10000 um2 (diam=sqrt(area), L = diam/pi). I tried reduced dimensions, but the basket cells develop unrealistic voltage traces (no spikes, highly depolarized, probably blocked) and E/I balance get lost. Connectivity weights and external stimulation should be tuned for new dimensions. However, if we reduce ONLY the diameter, the impact is very mild. I tried simulations with diam = 10 and L = 31.831 and look fine (the rhythms seem a bit more disorganized). For visualization, this could be more appealing.
